packages/editor/src/client/blocks/richTextBlockTable.less

```diff
@@ -5,10 +5,8 @@ table {
   table-layout: fixed;
 }
 
-table,
-td,
-tr {
-  border: 1px solid black;
+td {
```

Comment:
I guess you can use only one `td`-declaration instead of two here? (I meant you can include the `padding` declaration in here?)